content/en/docs/instrumentation/erlang/resources.md

```diff
@@ -9,16 +9,7 @@ cSpell:ignore: behaviour
 <!-- markdownlint-disable no-duplicate-heading -->
 <!-- markdownlint-capture -->
 
-A [resource](/docs/specs/otel/overview/#resources) represents an entity
-producing telemetry as attributes. For example, an OTP Release producing
-telemetry that is running in a container on Kubernetes has an OTP Release name,
-a Pod name, a namespace, and possibly a deployment name. All four of these
-attributes can be included in the resource.
-
-In your observability backend, you can use resource information to better
-investigate interesting behavior. For example, if your trace or metrics data
-indicate latency in your system, you can narrow it down to a specific container,
-pod, or Kubernetes deployment.
+{{% docs/instrumentation/resources-intro erlang %}}
```

Comment:
As an Erlang/Elixir Reviewer here, I guess this is okay, but it certainly is weird that the Erlang-specific term ("an OTP Release") ends up living in the generic-side of the documentation rather than being passed as another optional term—I notice that Go and Java docs don't need to pass the programming language in there either and there are no problems with that.

If you were to truly extract this bit to share it elsewhere, I'd argue you want to keep the language-specific terms on the caller side, and the OTel-general terms on the template's side.

Reply:
fair point, it felt odd doing it the way I did it here, so let me fix that.